index.html

```diff
@@ -17,11 +17,42 @@
 <body>
 
     <div class="container">
+      <nav class="navbar navbar-expand-lg navbar-light  justify-content-end">
+        <a class="navbar-brand" href="#">Portfolio</a>
+        <a class="nav-link" href="#">Contact</a>
+        <a class="nav-link" href="#">Blog</a>
+      </nav>
       <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
+          <div class="jumbotron col-7 justify-content-center">
+            <img class="img col-4" src="https://i.imgur.com/Uwl6plE.jpg" alt="unprofessional picture">
+            <h1 class="display-4 text-center">Harrison Crawford <span class="badge badge-info">CS8</span></h1>
+            <p class="lead">My specialty is Javascript.</p>
+            <hr class="my-4">
+            <a class="btn btn-danger"href="#">click here to die instantly</a>
           </div>
+          <ul class="list-group col-xs-">
```

Comment:
The pattern for Bootstrap grid system is this:

```
<div class="container">
  <div class="row">
    <div class="col">
       <ul></ul> <!-- content goes here -->
    </div>
    <div class="col">
      <img></img> <!-- content goes here -->
    </div>
  </div>
</div>
```

If you notice, there's one main parent container with class `container`, with `row` div separating the sections of your page, and `col` div separating the content within the `row` divs.